manager/manager.go

```diff
@@ -712,6 +729,7 @@ func (m *Manager) becomeLeader(ctx context.Context) {
 // becomeFollower shuts down the subsystems that are only run by the leader.
 func (m *Manager) becomeFollower() {
 	m.dispatcher.Stop()
+	m.logbroker.Stop()
```

Comment:
It looks like calling `m.logbroker.Stop` multiple times will panic. Since both `becomeFollower` and `(*Manager).Stop` can call `m.logbroker.Stop`, either logbroker's `Stop` function should protect against this (as I believe most `Stop` functions do), or the manager should set `m.logbroker` to `nil` after calling `Stop` and check before each use.